ecs-init/docker/docker.go

```diff
@@ -161,7 +161,7 @@ func (c *Client) loadEnvVariables() []string {
 
 func (c *Client) getHostConfig() *godocker.HostConfig {
 	binds := []string{
-		defaultDockerEndpoint + ":" + defaultDockerEndpoint,
+		dockerUnixSocket + ":" + dockerUnixSocket,
```

Comment:
Can you make sure that this gets mounted into the well-known location inside the container?

`config.DockerUnixSocket() + ":" + defaultDockerEndpoint`